idl/chromadb/proto/coordinator.proto

```diff
@@ -388,11 +388,23 @@ message GetCollectionSizeResponse {
 }
 
 message ListCollectionsToGcRequest {
-  // Only return collections with last GC time less than this value.
-  // This introduces a limit on the number of collections that can be returned.
-  // This allows for a cheap pagination, and also a way to deal with GC to
-  // garbage collect collections that have not been cleaned up for a long time.
-  // uint64 last_gc_time_lt = 1;
+  // Return collections that need to be GCed based on this cutoff time.
+  // Currently, sysdb will return all collections that have versions created
+  // before this cutoff time.
+  // SysDb can apply additional logic to return the collections that should
+  // be prioritized for GC.
+  optional uint64 cutoff_time_secs = 1;
```

Comment:
I think the sysdb should fail requests with a cutoff time of less than 30 mins from NOW() or similar as a backstop/defensive measure.